frontend_ta/src/components/MatchEntry.js

```diff
@@ -25,15 +25,22 @@ export class MatchEntry extends React.Component {
     
     componentDidMount(){
         if(sessionStorage.getItem("username")!=null){
-        getData("/api/match/current_match").then((response) => {
-            if (response.status === 200) {
-              this.setState({currentMatch: response.result,badResponse:null});
-              console.log(response.result);}
-             else this.setState({ badResponse: response.message });
-            }
-            );
-          }
+        // getData("/api/match/current_match").then((response) => {
+        //     if (response.status === 200) {
+        //       this.setState({currentMatch: response.result,badResponse:null});
+        //       console.log(response.result);}
+        //      else this.setState({ badResponse: response.message });
+        //     }
+        //     );
+        //   }
+          const data = [
```

Comment:
the old code is needed to play with the iframe(anyway this can be solved when you solve conclifts)